src/App.tsx

```diff
@@ -15,48 +16,117 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  SortByName = 'alphabet',
+  SortByLength = 'length',
+  DefaultOption = '',
+}
+
+const {
+  SortByName,
+  SortByLength,
+  DefaultOption,
+} = SortType;
```

Comment:
Better not to use destructuring here, will be more clear if you will use `SortType.ByName` for example